### PR TITLE
Title: #171792784 feedbacks 

### PR DESCRIPTION
**Pull request by DeliceLydia**
#### What does this PR do?
This PR is for feedbacks given last week and tests

#### Description of Task to be completed
 -worked on the feedbacks of the UI
 -started writing tests
 -added travis.yml file
 -wrote signup tests
 -wrote signin tests

#### How should this be manually tested?
-clone the project in your local machine
-cd in the project
-run npm install to update all packages used
-run index.html if you want to view UI(user interface)
-run npm start if you want to test endpoints on postman
-run npm test if you want to see the coverage

 #### Any background context you want to provide?
N/A

#### What are the relevant pivotal tracker stories?
(https://www.pivotaltracker.com/story/show/171792784)[#171792784]

#### Screenshots (if appropriate)
Screen Shot 2020-03-17 at 13 05 02

### CODE REVIEW

File: Server/tests/tests.js [hunk 1]
```diff
@@ -0,0 +1,171 @@
...
+    it('user should not be able to signin when provided unexisted email',(done)=>{
+        const returningUser = {
+            email: 'lydie@gmail.com',
+            password: 'Lydie1@'
+         };
+         chai.request(app)
+     .post('/api/v1/auth/signin')
+     .send(returningUser)
+     .end((err, res)=>{
+        expect(res.status).to.be.eql(406);
```
Comment (alainburindi) [inline Server/tests/tests.js]: What do you think about 406 status code for wrong input?? this one is rarely used and most of the time the message should include options for the user to choose
https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/406

**Review by alainburindi — CHANGES_REQUESTED** [review]
Well done but some pieces of feedback though
- next time try to avoid having many things done in one PR. for instance, in this case, the work is done both on the UI and the backend
- Would you mind removing the `.DS_Store` file and add it to `.gitignore`
- Try to fix the test's failure on travis-ci
- there is another comment about the `406` status code

File: Server/tests/tests.js (hunk 1 above)
Comment (DeliceLydia) [reply Server/tests/tests.js]: ooh okey according to the explanation status code is wrong..can it be a bad request 400?

Comment (alainburindi) [reply Server/tests/tests.js]: That one works

**Review by alainburindi — APPROVED** [review]
LGTM

**Pull request merged**